editor/components/inserter/index.js

```diff
@@ -20,6 +20,10 @@ class Inserter extends wp.element.Component {
 	}
 
 	toggle() {
+		if ( this.state.opened ) {
+			this.toggleNode.focus();
```

Comment:
@aduth 😑  I mean React doesn't have a native, easy, way to handle things like this one. To my understanding, you all don't like the way you had to implement this focus management, no?